apps/api/src/auth/oslo-auth.ts

```
@@ -0,0 +1,103 @@
+
+export function setSessionTokenCookie(c: Context<AppContext>, token: string, expiresAt: number): void {
+  if (env(c).WORKER_ENV === "production") {
+    c.res.headers.set("Set-Cookie", `session=${token}; HttpOnly; SameSite=Lax; Max-Age=${expiresAt}; Path=/; Secure;`);
+  } else {
+    c.res.headers.set("Set-Cookie", `session=${token}; HttpOnly; SameSite=Lax; Max-Age=${expiresAt}; Path=/`);
+  }
+}
+
+export function deleteSessionTokenCookie(c: Context<AppContext>): void {
+  if (env(c).WORKER_ENV === "production") {
+    c.res.headers.set("Set-Cookie", `session=; HttpOnly; SameSite=Lax; Max-Age=0; Path=/; Secure;`);
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)